src/DIRAC/Core/Tornado/Server/TornadoServer.py

```diff
@@ -60,35 +60,36 @@ class TornadoServer(object):
 
     Example 2:We want to debug service1 and service2 only, and use another port for that ::
 
-      services = ['component/service1', 'component/service2']
-      serverToLaunch = TornadoServer(services=services, port=1234)
+      services = ['component/service1:port1', 'component/service2']
+      endpoints = ['component/endpoint1:port1', 'component/endpoint2']
```

Comment:
that is wrong, the different endpoints cannot have different ports

Reply:
fixed